chrome/content/zotero/integration/integrationDocPrefs.xul

```diff
@@ -77,4 +84,4 @@
 		<checkbox id="storeReferences" label="&zotero.integration.prefs.storeReferences.label;"/>
 		<description class="radioDescription">&zotero.integration.prefs.storeReferences.caption;</description>
 	</vbox>
-</dialog>
\ No newline at end of file
+</dialog>
```

Comment:
(You can do that easily with `git checkout -p chrome/content/zotero/integration/integrationDocPrefs.xul`.)